src/mcode-pilot/components/TreatmentOptionsOutcomes/TreatmentOptionsOutcomes.jsx

```diff
@@ -134,9 +134,14 @@ export default class TreatmentOptionsOutcomes extends Component {
 
     generateRow(patients, treatments) {
         if (patients.length === 0) return [];
-
-        const treatmentName = _.isArray(treatments) ? treatments.map(name => TREATMENT_NAMES[name]).join(' & ') : TREATMENT_NAMES[treatments];
-        let row = this.initializeRow(treatmentName);
+        let displayName = _.isArray(treatments) ? 
+            treatments === this.state.includedTreatments ?
```

Comment:
Are we looking for deep equality here? This will only return true if the variables are pointing to the same array; said another way, it will return false even if they have the same contents. If we want a shallow equality, we can use Lang.isEqual.

Reply:
deep equality is not needed, in this case we actually want to check that they reference the same array, not the content.

Reply:
Just wanted to check!